
Xvnc Free Edition 4.1.1 - built Feb 25 2015 23:02:21
Copyright (C) 2002-2005 RealVNC Ltd.
See http://www.realvnc.com for information on VNC.
Underlying X server release 40300000, The XFree86 Project, Inc


Thu Feb 13 09:47:00 2020
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on port 5907
 vncext:      created VNC server for screen 0
error opening security policy file /etc/X11/xserver/SecurityPolicy
Could not init font path element /usr/X11R6/lib/X11/fonts/Type1/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/Speedo/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/misc/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/75dpi/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/100dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/75dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/100dpi/, removing from list!
xrdb: No such file or directory
xrdb: can't open file '/home/is28/kruglov28/.Xresources'
/usr/bin/startxfce4: X server already running on display :7
gpg-agent[6197]: WARNING: "--write-env-file" is an obsolete option - it has no effect
gpg-agent: a gpg-agent is already running - not starting a new one

(xfce4-session:6187): xfce4-session-WARNING **: 09:47:03.729: gpg-agent returned no PID in the variables

(xfce4-session:6187): xfce4-session-WARNING **: 09:47:03.731: xfsm_manager_load_session: Something wrong with /home/is28/kruglov28/.cache/sessions/xfce4-session-virt-1:7, Does it exist? Permissions issue?

(xfwm4:6201): xfwm4-WARNING **: 09:47:03.757: The display does not support the XRender extension.

(xfwm4:6201): xfwm4-WARNING **: 09:47:03.757: The display does not support the XComposite extension.

(xfwm4:6201): xfwm4-WARNING **: 09:47:03.757: The display does not support the XDamage extension.

(xfwm4:6201): xfwm4-WARNING **: 09:47:03.757: The display does not support the XFixes extension.

(xfwm4:6201): xfwm4-WARNING **: 09:47:03.757: Compositing manager disabled.

** (light-locker:6211): ERROR **: 09:47:03.791: Environment variable XDG_SESSION_PATH not set. Is LightDM running?

(polkit-gnome-authentication-agent-1:6213): GLib-CRITICAL **: 09:47:03.798: g_variant_new_string: assertion 'string != NULL' failed

(polkit-gnome-authentication-agent-1:6213): polkit-gnome-1-WARNING **: 09:47:03.799: Failed to register client: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files
xfwm4-Message: 09:47:03.823: Unsupported keyboard modifier '<Super>Tab'
(uint32 1,)

(xfsettingsd:6210): xfsettingsd-CRITICAL **: 09:47:03.883: RANDR extension is too old, version 1.1. Display settings won't be applied.
Xlib:  extension "XInputExtension" missing on display ":7.0".

(xfsettingsd:6210): xfsettingsd-CRITICAL **: 09:47:03.883: XI is not present.

(xfsettingsd:6210): xfsettingsd-CRITICAL **: 09:47:03.884: Failed to initialize the Xkb extension.

(xfsettingsd:6210): xfsettingsd-CRITICAL **: 09:47:03.892: Failed to initialize the Accessibility extension.
xfwm4-Message: 09:47:03.930: Unsupported keyboard modifier '<Super>Tab'

(nm-applet:6318): Gtk-WARNING **: 09:47:04.056: Can't set a parent on widget which has a parent

(nm-applet:6318): Gdk-CRITICAL **: 09:47:04.195: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
/usr/lib/python3/dist-packages/blueman/plugins/applet/AppIndicator.py:8: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3 as girAppIndicator
ERROR:dbus.proxies:Introspect error on org.bluez:/org/bluez: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.268" (uid=1274 pid=6278 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.268" (uid=1274 pid=6278 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.268" (uid=1274 pid=6278 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)

(wrapper-1.0:6363): Gdk-WARNING **: 09:47:08.383: /build/gtk+2.0-AoeliP/gtk+2.0-2.24.32/gdk/x11/gdkdrawable-x11.c:952 drawable is not a pixmap or window

(wrapper-1.0:6363): Gdk-WARNING **: 09:47:08.383: /build/gtk+2.0-AoeliP/gtk+2.0-2.24.32/gdk/x11/gdkdrawable-x11.c:952 drawable is not a pixmap or window

Thu Feb 13 09:48:40 2020
 Connections: accepted: 0.0.0.0::49708
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)

Thu Feb 13 09:48:43 2020
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565

(Thunar:6207): Gdk-WARNING **: 09:50:53.846: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 09:51:02.219: gdk_window_set_icon_list: icons too large

(polkit-gnome-authentication-agent-1:6213): polkit-gnome-1-WARNING **: 09:54:42.408: Couldn't open user icon: Failed to open file '/home/adom/.face': No such file or directory

(polkit-gnome-authentication-agent-1:6213): polkit-gnome-1-WARNING **: 09:54:42.409: Couldn't open user icon: Failed to open file '/home/test/.face': No such file or directory
Gtk-Message: 09:54:42.437: GtkDialog mapped without a transient parent. This is discouraged.

(Thunar:6207): GLib-GIO-CRITICAL **: 09:54:44.349: g_file_hash: assertion 'G_IS_FILE (file)' failed

(Thunar:6207): GLib-GIO-CRITICAL **: 09:54:44.349: g_file_query_info_async: assertion 'G_IS_FILE (file)' failed

(Thunar:6207): GLib-GObject-CRITICAL **: 09:54:44.349: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(Thunar:6207): Gdk-WARNING **: 09:54:46.116: gdk_window_set_icon_list: icons too large

Thu Feb 13 10:44:49 2020
 Connections: closed: 0.0.0.0::49708 (Clean disconnection)
 SMsgWriter:  framebuffer updates 36267
 SMsgWriter:    copyRect rects 378, bytes 6048
 SMsgWriter:    hextile rects 58544, bytes 67309042
 SMsgWriter:    raw bytes equivalent 631347948, compression ratio 9.379839

(Thunar:6207): Gdk-WARNING **: 11:41:11.068: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 06:44:17.037: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 06:10:41.071: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 12:04:41.078: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 12:07:59.033: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 12:08:17.091: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 12:14:11.063: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 12:14:29.059: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 12:17:29.053: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 13:11:05.083: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 13:06:59.122: gdk_window_set_icon_list: icons too large

(Thunar:6207): Gdk-WARNING **: 13:07:59.123: gdk_window_set_icon_list: icons too large
/usr/lib/ubuntu-release-upgrader/check-new-release-gtk:30: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk
WARNING:root:timeout reached, exiting

(xfdesktop:6209): libxfce4ui-WARNING **: 09:42:19.910: ICE I/O Error

(xfdesktop:6209): libxfce4ui-WARNING **: 09:42:21.917: Disconnected from session manager.
xfwm4: Fatal IO error 4 (Interrupted system call) on X server :7.0.
xfdesktop: Fatal IO error 25 (Inappropriate ioctl for device) on X server :7.0.
